src/utils/config_loader.ts

```diff
@@ -0,0 +1,10 @@
+import { get } from 'lodash';
+import { CustomConfig, WidgetConfig } from '../interfaces/custom_config';
+
+export class ConfigLoader {
```

Comment:
IMO, `AppConfigLoader` would be better.
BTW, prefer dashed-case for file names.